src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/hosts.po.ts

```diff
@@ -73,6 +74,31 @@ export class HostsPageHelper extends PageHelper {
     });
   }
 
+  @PageHelper.restrictTo(pages.index.url)
+  maintenance(hostname: string, exit = false) {
+    this.getTableCell(this.columnIndex.hostname, hostname).click();
+    this.clickActionButton('enter-maintenance-mode');
+
+    this.getTableCell(this.columnIndex.hostname, hostname)
+      .parent()
+      .find(`datatable-body-cell:nth-child(${this.columnIndex.status})`)
+      .should(() => {
+        cy.get('.badge', { timeout: 25000 }).contains('maintenance');
+      });
+
+    if (exit) {
+      this.getTableCell(this.columnIndex.hostname, hostname).click();
+      this.clickActionButton('exit-maintenance-mode');
+
+      this.getTableCell(this.columnIndex.hostname, hostname)
+        .parent()
+        .find(`datatable-body-cell:nth-child(${this.columnIndex.status})`)
+        .should(() => {
+          cy.get('.badge', { timeout: 15000 }).contains('');
```

Comment:
`25000` `15000`... I guess it's milliseconds, right? Where do all these timeout come from? Try and error? In any case I think it's better to have more homogeneous timeouts (let's go for the longer one) and replace them with global consts, instead of tuning them for each query.

Reply:
Updated the tests.